### DISCUSSION TITLE
Transcription File is Blank

### DISCUSSION THREAD

**Discussion by rjdunne** [thuliteio/doks#1315] [General]
I uploaded an MP3 file for transcription and I receive notification the process was complete.  The email I received was a link to a blank DOCX file.  Any ideas?

**Comment by h-enk** [comment]
Did the transcription process output the file to the `public` directory?

**Reply by h-enk** [reply]
Could be that the docx file is corrupted (not processed correctly). Why not generate a pdf?

**Comment by rjdunne** [comment]
This is what I get when I click on the link I was emailed saying the work
was complete.

This did not go to a public directory but was on your server.  When I tried
to download or open the files were blank.

**Comment by rjdunne** [comment]
The PDF file could not open.   Can I try to upload the file again without
being charged?

**Reply by h-enk** [reply]
Doks is a just a (static) documentation theme. It looks like you're using a transcription service that is using Doks...

I'm curious, what's the name/ website of this service?

**Comment by rjdunne** [comment]
https://speechnotes.co/transcribe/

**Reply by h-enk** [reply]
Raise an issue or open a discussion here: https://github.com/speechnotes/speechnotes-website

Or, email the contact: https://github.com/speechnotes

**Comment by rjdunne** [comment]
Thanks
